app/src/main/res/values/colors.xml

```diff
@@ -15,4 +16,5 @@
     <color name="color_deep_purple">#1b0143</color>
     <color name="color_little_purple">#5c2ba9</color>
     <color name="color_dark_opacity">#64000000</color>
+    <color name="color_mini_opacity">#b4000000</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_mini_opacity` appears to be unused
 Unused resources make applications larger and slow down builds.